### ISSUE TITLE
Placing Frame Chest on Hopper connected to Drawers Controller Crashes Server

### ISSUE THREAD

**Issue by SaphieNyako** [PianoManu/BlockCarpentry#32]
Hello!

We tried placing a frame chest on a hopper that is connected to a Create system but it keeps crashing the server.
It is also connected to a Drawers Controller from Storage Drawers. In the picture you can see the setup. 
When placing it on a hopper alone nothing happens, its specifically this system.
The red controller is from Mystical agriculture connecting it with that didnt do it either.
But when connecting it with the Drawer Controller from Storage Drawers it crashes. 

[crash-2021-02-22_09.20.45-server.txt](https://github.com/PianoManu/BlockCarpentry/files/6021333/crash-2021-02-22_09.20.45-server.txt)
[crash-2021-02-22_09.19.44-server.txt](https://github.com/PianoManu/BlockCarpentry/files/6021334/crash-2021-02-22_09.19.44-server.txt)
![2021-02-22_11 36 56](https://user-images.githubusercontent.com/77398437/108697125-b9a64c80-7502-11eb-8bcf-b9925442a55a.png)

**Comment by PianoManu** [comment]
First of all, sorry for the late response, I've had a rough time lately.
I just read the crash report and I guess the problem is with hoppers, that they can't extract stuff from framed chests... I'll look into this issue further. Thanks for reporting!

**Comment by PianoManu** [comment]
I don't exactly know how you set up the system - I took a frame chest and connected it via hoppers to a drawer controller, and I received the same crash like you. The funny thing is, that it worked, when I changed the tile entity class to extend the vanilla chest tile entity. I can't recreate that error anymore. I'll release the fixed version today in the afternoon or tomorrow (0.3.5b). Thanks for reporting and sorry for the circumstances
